images/Dockerfile.opensuse

```diff
@@ -32,7 +32,7 @@ RUN zypper ar -G https://download.opensuse.org/repositories/utilities/openSUSE_F
     zypper ref
 
 FROM ${FLAVOR_RELEASE}-repo AS common
-RUN zypper in --force-resolution -y \
+RUN zypper in -y \
```

Comment:
what's new?

Reply:
oops, I was just debugging an issue and forgot to remove